orchestrator.dockerfile

```
@@ -1,14 +1,14 @@
-FROM cogment/orchestrator:v1.0.3
+FROM cogment/orchestrator:v2.0.0
 
 RUN apt-get update && apt-get -y install gettext-base
 
 ADD cogment.yaml ./
 ADD *.proto ./
 
-ENV TRIAL_LIFECYCLE_PORT=9000
-ENV TRIAL_ACTOR_PORT=9000
+ENV COGMENT_LIFECYCLE_PORT=9000
+ENV COGMENT_ACTOR_PORT=9000
 #ENV PROMETHEUS_PORT=8000
 
 # Didn't manage to setup envsubst in the ENTRYPOINT
 ENTRYPOINT []
-CMD envsubst < /app/cogment.yaml > /app/cogment.yaml.out && orchestrator --config=/app/cogment.yaml.out
+CMD envsubst < /app/cogment.yaml > /app/cogment.yaml.out && orchestrator --config=/app/cogment.yaml.out --params=/app/cogment.yaml.out
```

```suggestion
CMD envsubst < /app/cogment.yaml > /app/cogment.yaml.out && orchestrator --params=/app/cogment.yaml.out
```

And I'm not sure you'll even need it if you move the datastore definition to the prehook